webapp/src/tutorial.tsx

```diff
@@ -100,13 +103,21 @@ function getUsedBlocksInternalAsync(code: string[], id: string, language?: strin
                             usedBlocks[blk.type] = 1;
                         }
 
-                        const comment = blk.getCommentText();
+                        let comment = blk.getCommentText();
                         if (comment && /@highlight/.test(comment)) {
                             if (!highlightBlocks[snippetHash][blk.type]) {
                                 highlightBlocks[snippetHash][blk.type] = 0;
                             }
                             highlightBlocks[snippetHash][blk.type] = highlightBlocks[snippetHash][blk.type] + 1;
                         }
+                        while (comment && /@validate-\S+/.test(comment)) {
+                            const marker = comment.match(/@validate-(\S+)/)[1];
+                            comment = comment.replace(/@validate-\S+/, "");
```

Comment:
I think cutting out just the `@` would be fine (so it's in there as key `validate-exists` instead of just `exists`) -- avoids us having to remember that the value changes slightly from markdown -> map, and also makes it so we don't have any weirdness if we need to track other meta fields later on (e.g. including highlight)

Reply:
Actually just thinking on this again, would `check-` as the prefix maybe be easier to read (so like `check-exists` / `check-sequence` / ...)? `validate` feels mildly fancy / technical to me. This doesn't matter much / we can defer the decision till a bit later if we want since it'll be a little while before we start shipping this code / markdown making use of it

Reply:
I'll just leave validate for now, but I did change this regex match to basically grab all the keys in the comment regardless of the prefix. So it's getting highlight now too and I can build on that in a future PR.